DrinkManagerWeb/Views/Admin/CreateUser.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model DrinkManagerWeb.Models.ViewModels.UserViewModel
+@{
+    ViewBag.Title = "Administration - Users";
+    Layout = "_LayoutAdmin";
+}
+
+<h2>Create User</h2>
+<hr />
+<form method="post">
```

Comment:
i Would add validation summary here in the form

    <div asp-validation-summary="All" class="text-danger"></div>

Reply:
Done with change of "All" value to "ModelOnly" so we can see there only model-level error messages (and don't show individual property messages, as they are displayed under each property field).